### PR TITLE
Initial Commit of Wishlist Page

### PR DESCRIPTION
**Pull request by laxmikanthkonthum**
You can ignore other files as I just added links to them.
Deploy link: https://deploy-preview-4--objective-hugle-0f76fc.netlify.app/wishlist-page/wishlist.html

### CODE REVIEW

File: wishlist-page/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Laxmikanth Konthum" />
+    <meta name="description" content="Elite-Ecom is a E-commerce website created using Elite-UI" />
+    <title>Wishlist</title>
+    <link href="wishlist.css" rel="stylesheet">
+</head>
+
+<body>
+    <nav class="nav-bar nav-colored top">
+
```
Comment (aniketsingh13) [inline wishlist-page/wishlist.html:16]: remove space

File: wishlist-page/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                </button>
+            </div>
+        </div>
+    </main>
+    <footer class="footer bottom-left">
+        <p>A E-com site for food lovers built using <a class="link-plain text-bold"
+                href="https://elite-ui.netlify.app/">Elite-UI</a></p>
+    </footer>
+</body>
+
```
Comment (aniketsingh13) [inline wishlist-page/wishlist.html:140]: here also

**Review by aniketsingh13 — APPROVED** [review]
look good to  me

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me

File: wishlist-page/wishlist.html (hunk 1 above)
Comment (laxmikanthkonthum) [reply wishlist-page/wishlist.html:16]: Beautify is doing it

**Pull request merged**